bundles/org.eclipse.passage.lic.base/src/org/eclipse/passage/lic/base/io/FileCollection.java

```diff
@@ -43,34 +42,20 @@ public FileCollection(Supplier<Path> base, PassageFileExtension extension) {
 	}
 
 	public Collection<Path> get() throws LicensingException {
-		HunterFiles hunter = new HunterFiles(extensions);
-		try {
-			Files.walkFileTree(base.get(), hunter);
+		try (Stream<Path> files = filesIn(base.get())) {
+			return collectFiles(files);
```

Comment:
Dear @HannesWell we are serious about query/action naming convention :) This one is a _query_ too, should be named with proper noun.

Reply:
I never questioned that you are serious. :) I'm not used to this style and therefore oversaw this. Sorry for that.
Is the `codecov`-check helping in this regard? I have to admit that I don't understand its output.

Reply:
`codecov` now running with settings close to default due to lack of dev resources, will find out how to configure it eventually.

Reply:
No problem.
I was just wondering if I would save your time to review this change if I were able to understand its output.
If it is not configured this does not help in this regard.